src/main/rest-api/idGenerator.test.js

```diff
@@ -0,0 +1,15 @@
+import { idGenerator } from './idGenerator';
+
+describe('Verify that IdGenerator class', () => {
```

Comment:
Maybe change the description because it's not a class anymore. But the rest seems fine to me 👍🏻